src/components/dropdown/Dropdown.spec.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Dropdown } from "./Dropdown";
+
+const TEST_LABEL = "Option 1";
+const SEARCH_VALUE = "Option";
+
+function DropdownTestComponent() {
+  const options = [TEST_LABEL];
+  const [value, setValue] = React.useState("");
+  return (
+    <Dropdown
+      name="Select an option"
+      options={options}
+      value={value}
+      width="15rem"
+      minNumOfOptionsToShowSearchBox={4}
+      onChange={(val) => setValue(val)}
+      title="Example title"
+    />
+  );
+}
+
+const getSearchComponentAndReturnSearchBox = () => {
+  const options = [
+    { value: "test", label: TEST_LABEL },
+    { value: "Option 2" },
+    "Option 3",
+    "Option 4",
+  ];
+
+  render(
+    <Dropdown
+      name="Select an option"
+      options={options}
+      width="15rem"
+      minNumOfOptionsToShowSearchBox={4}
+      onChange={() => {}}
+      title="Example title"
+    />
+  );
+
+  userEvent.click(screen.getByRole("button"));
+  return screen.getByRole("searchbox");
+};
```

Comment:
I think we could reuse `DropdownTestComponent`
```suggestion
function DropdownTestComponent() {
   const options = [
     { value: "test", label: TEST_LABEL },
     { value: "Option 2" },
     "Option 3",
     "Option 4",
  ];
  const [value, setValue] = React.useState("");
  
  return (
    <Dropdown
      name="Select an option"
      options={options}
      value={value}
      width="15rem"
      minNumOfOptionsToShowSearchBox={4}
      onChange={(val) => setValue(val)}
      title="Example title"
    />
  );
}

const getSearchComponentAndReturnSearchBox = () => {
  render(
    <DropdownTestComponent />
  );

  userEvent.click(screen.getByRole("button"));
  return screen.getByRole("searchbox");
};
```

Reply:
Done, thanks :D